src/Raven.Server/Documents/Versioning/VersioningStorage.cs

```diff
@@ -17,6 +18,9 @@ namespace Raven.Server.Documents.Versioning
 {
     public unsafe class VersioningStorage
     {
+        public static readonly Slice KeyAndEtag = Slice.From(StorageEnvironment.LabelsContext, "KeyAndEtag", ByteStringType.Immutable);
```

Comment:
Use the "Slices" on the name for static slices that are created on LabelsContext.